omod/pom.xml

```diff
@@ -125,16 +125,6 @@
             <artifactId>htmlformentry-api-2.0</artifactId>
         </dependency>
 
-		<dependency>
```

Comment:
@faithNassiwa Leave these dependencies they are required for the tests, what issues are they causing?

Reply:
@ssmusoke I got these warnings and initially, I had been getting maven related errors which I had thought were due to duplicate dependencies. 
```console
INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.openmrs.module:aijar-omod:jar:2.0.1-SNAPSHOT
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.openmrs.module:htmlformentry-api-2.0:jar -> duplicate declaration of version (?) @ org.openmrs.module:aijar-omod:[unknown-version], /Users/faith/openmrs/UgandaEMR/watched-projects/aijar/omod/pom.xml, line 128, column 15
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 

```

Reply:
@faithNassiwa I will note that on the issues board and fix it - but should not cause any problems.